.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+*.iml
+.idea
```

Comment:
I honestly don't mind to have those files on gitignore, despite being specific to an IDE. In theory we should add to our local gitignore.
Anyway, the last line is missing a line break.

Reply:
Agree. We usually add these to our global .gitignore file.